backend/src/test/java/gov/cdc/usds/simplereport/api/ApiUserManagementTest.java

```diff
@@ -413,6 +412,26 @@ void updateUserRole_orgUser_failure() {
     runQuery("update-user-role", updateRoleVariables, ACCESS_ERROR);
   }
 
+  @Test
+  void updateUserRole_orgAdmin_roleUpdated() {
+    useOrgEntryOnly();
+    ObjectNode who = (ObjectNode) runQuery("current-user-query").get("whoami");
+    assertEquals("Test-entry user", who.get("roleDescription").asText());
+    String id = who.get("id").asText();
+
+    useOrgAdmin();
+    ObjectNode updateRoleVariables =
+        JsonNodeFactory.instance
+            .objectNode()
+            .put("id", id)
+            .put("role", OrganizationRole.ADMIN.name());
+    runQuery("update-user-role", updateRoleVariables);
+
+    useOrgEntryOnly();
+    who = (ObjectNode) runQuery("current-user-query").get("whoami");
+    assertEquals("Admin user", who.get("roleDescription").asText());
```

Comment:
well this certainly strikes me as odd but i have no better solution

Reply:
Not sure what you mean? This just seemed like the most convenient way of checking that I'd successfully updated the user's roles—if it seems too gross, I'm happy to change it to something cleaner!

Reply:
eh, something about how it should _never_ be true that a user derived from calling `useOrgEntryOnly()` returns a role description of "Admin user", and how that means we should change the name of `useOrgEntryOnly()`. but honestly i think that would cause more problems than it would solve, so never mind.